hermes/Cargo.toml

```diff
@@ -51,32 +37,50 @@ libp2p                         = { version = "0.42.2", features = [
     "yamux",
 ]}
 
-async-trait = "0.1.68"
+log                    = { version = "0.4.17" }
+moka                   = { version = "0.11.0", features = ["future"] }
+pyth-sdk               = { version = "0.7.0" }
 
-# We around bound to this version because of pyth-oracle
-solana-client = "=1.13.3"
-solana-sdk = "=1.13.3"
-solana-account-decoder = "=1.13.3"
+# Parse Wormhole attester price attestations.
+pythnet-sdk            = { path = "../pythnet/pythnet_sdk/", version = "=1.13.6" }
 
-moka = { version = "0.11.0", features = ["future"] }
-byteorder = "1.4.3"
-serde_qs = { version = "0.12.0", features = ["axum"] }
+rand                   = { version = "0.8.5" }
+reqwest                = { version = "0.11.14", features = ["blocking", "json"] }
+ring                   = { version = "0.16.20" }
+rusqlite               = { version = "0.28.0", features = ["bundled"] }
+secp256k1              = { version = "0.26.0", features = ["rand", "recovery", "serde"] }
+serde                  = { version = "1.0.152", features = ["derive"] }
+serde_arrays           = { version = "0.1.0" }
+serde_cbor             = { version = "0.11.2" }
+serde_json             = { version = "1.0.93" }
+serde_qs               = { version = "0.12.0", features = ["axum"] }
+serde_wormhole         = { git = "https://github.com/wormhole-foundation/wormhole", tag = "v2.17.1" }
+sha256                 = { version = "1.1.2" }
+sha3                   = { version = "0.10.4" }
 
-serde_wormhole = { git = "https://github.com/wormhole-foundation/wormhole", tag = "v2.17.1"}
-wormhole-sdk = { git = "https://github.com/wormhole-foundation/wormhole", tag = "v2.17.1" }
-pyth-oracle = { git = "https://github.com/pyth-network/pyth-client", rev = "1bdac0d2ee39a5a9f27eb86e4fd01229bd68aa5a" , features = ["library", "strum", "serde"] }
+# We around bound to this version because of pyth-oracle
+solana-client          = { version = "=1.13.3" }
+solana-sdk             = { version = "=1.13.3" }
+solana-account-decoder = { version = "=1.13.3" }
 
+structopt              = { version = "0.3.26" }
+strum                  = { version = "0.24.1", features = ["derive"] }
+tokio                  = { version = "1.26.0", features = ["full"] }
+tower-http             = { version = "0.4.0", features = ["cors"] }
+typescript-type-def    = { version = "0.5.5" }
+wormhole-sdk           = { git = "https://github.com/wormhole-foundation/wormhole", tag = "v2.17.1" }
 
-strum = { version = "0.24", features = ["derive"] }
-ethabi = { version = "18.0.0", features = ["serde"] }
-sha3 = "0.10.4"
-humantime = "2.1.0"
+# Oracle message types
+[dependencies.pyth-oracle]
+git = "https://github.com/pyth-network/pyth-client"
+rev = "1bdac0d2ee39a5a9f27eb86e4fd01229bd68aa5a"
+features = ["library", "strum", "serde"]
```

Comment:
This is ugly, but i'm gonna remove it in a future PR to use the types that are moved to pythnet